packages/connect-form/src/components/advanced-options-tabs/advanced-options-tabs.tsx

```diff
@@ -63,8 +95,28 @@ function AdvancedOptionsTabs({
       {tabs.map((tabObject: TabObject, idx: number) => {
         const TabComponent = tabObject.component;
 
+        const tabHasError = !!errors.find(
```

Comment:
suggestion: there are functions to get errors by id, it would be nice to keep that convention and extract functions for getting errors/warnings by tabs. I doubt we will ever change the data structure for errors and warnings but is good to have the methods that manipulate it all in the same place.